package.json

```diff
@@ -36,14 +36,14 @@
   },
   "dependencies": {
     "@blackbaud/skyux-builder-plugin-stache": "~1.3.0",
+    "@blackbaud/skyux-builder-stache-search": "^1.3.0",
```

Comment:
Just to be clear: we want every stache site to have search installed, correct?

Reply:
Yep!